.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,5 @@
+- With issues:
+  - Use the search tool before opening a new issue.
+  - Please provide source code and commit sha if you found a bug.
+  - Review existing issues and provide feedback or react to them.
+
```

Comment:
maybe we can following the issue template: https://github.com/go-gitea/gitea/blob/master/.github/issue_template.md

* gin version (or commit ref):

Reply:
nice, I will update it. thanks!

Reply:
@appleboy done.